src/control/README.md

```diff
@@ -1,113 +1,41 @@
+Specific library packages can be found in lib/ which provide access
+to native storage libraries through language bindings e.g. lib/spdk
+or specific formatting capabilities e.g. lib/hostlist or lib/txtfmt.
 
-The [management tool](cmd/dmg) is an example client application which can connect to both the [agent](cmd/agent) to perform security functions (such as providing credentials and retrieving security contexts) and to the local management server to perform management functions (such as storage device discovery).
+The pbin package provides framework for forwarding of requests to be
+executed by the privileged binary `daos_admin` on behalf of `daos_server`.
 
-## Documentation
+The provider package contains interface shims to the external environment,
+initially just to the Linux operating system.
 
-- [Management API](https://godoc.org/github.com/daos-stack/daos/src/control/client)
-- [Management internals](https://godoc.org/github.com/daos-stack/daos/src/control/server)
-- [Agent API](https://godoc.org/github.com/daos-stack/daos/src/control/client/agent)
-- [Agent internals](https://godoc.org/github.com/daos-stack/daos/src/control/security)
-- [dRPC](https://godoc.org/github.com/daos-stack/daos/src/control/drpc)
-- [server package](server/README.md)
-- [management tool package](cmd/dmg/README.md)
-- [client package](client/README.md)
-- [common package](common/README.md)
+The system package encapsulates the concept of the DAOS system and
+it's associated membership.
 
-## Architecture
+## Developer Documentation
 
-First a view of software component architecture:
+Please refer to package specific README's.
 
-![Architecture diagram](/doc/graph/system_architecture.png)
+https://godoc.org/github.com/daos-stack/daos/src/control
 
-There are operations that will be performed on individual nodes in parallel, such as hardware provisioning, which will execute through storage or network libraries.
+## User Documentation
 
-Such broadcast commands (which will connect to a list of hosts) will usually be issued by the [management tool](dmg), a gRPC client, and handled by the gRPC [MgmtCtlServer](server/mgmt.go) running in `daos_server`.
-These commands will not traverse dRPC but will perform node-local functions such as hardware (network and storage) provisioning.
-
-Other operations which will interact with the DAOS data plane through the replicated management service.
-Such operations will be triggered via a single control plane instance (access point) and use the replicated management service (running in the data plane) to perform a distributed operation such as creating a storage pool.
-
-Commands which require connection to an access point will be forwarded to the data plane ([iosrv](/src/iosrv)), redirected by gRPC [MgmtSvc](server/mgmt_svc.go) over dRPC channel and handled by the [mgmt](/src/mgmt/srv.c) module.
-
-## Development Requirements
```

Comment:
Do we not want any of this dev info anymore?

Reply:
this is been kept up-to-date in the online documentation under "development"